kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -518,9 +532,6 @@ define([
                 const jobId = this.model.getItem('exec.jobState.job_id');
                 if (jobId) {
                     this.bus.emit(jcm.MESSAGE_TYPE.CANCEL, { [jcm.PARAM.JOB_ID]: jobId });
-                    if (this.looper) {
-                        this.looper.clearRequest();
-                    }
```

Comment:
this will get removed anyway when the next job status message comes in, so no need to delete it here